src/main/target/SITL/target.c

```diff
@@ -577,11 +583,18 @@ FLASH_Status FLASH_ProgramWord(uintptr_t addr, uint32_t value) {
 void uartPinConfigure(const serialPinConfig_t *pSerialPinConfig)
 {
     UNUSED(pSerialPinConfig);
-    printf("uartPinConfigure");
+    printf("uartPinConfigure\n");
+}
+
+void IOConfigGPIO(IO_t io, ioConfig_t cfg)
+{
+    UNUSED(io);
+    UNUSED(cfg);
+    printf("IOConfigGPIO\n");
```

Comment:
alternatively, it's possible to add undef `USE_PINIO, USE_PINIOBOX, USE_PIN_PULL_UP_DOWN` (see https://github.com/betaflight/betaflight/pull/9352/files#diff-f682e0332717b305936b703833517366)

Reply:
I am surprised that this even built before these stubs were added - are we missing some compiler settings to make this fail for the default build?

Reply:
@mikeller for me it didn't build without my change - I think it just hasn't been attempted to build in a while.

Reply:
It is built as part of our CI on Travis and on Jenkins (see https://ci.betaflight.tech/job/Betaflight/2018/artifact/obj/betaflight_4.2.0_SITL-2018-6433aaa68.hex). Unfortunately we do not have strong checks in place for the compiler suite version used for this target, so I suspect the build problems are gcc version dependent.

Reply:
> I am surprised that this even built before these stubs were added - are we missing some compiler settings to make this fail for the default build?

It builds with make TARGET=SITL, but fails with make TARGET=SITL DEBUG=GDB. I think that in release build some of PINIO-related calls is optimized away.